Files/Views/Pages/Properties.xaml

```diff
@@ -152,21 +154,24 @@
                     Grid.Column="0"
                     FontWeight="SemiBold"
                     Text="MD5Hash:"
-                    Visibility="{x:Bind ItemProperties.ItemMD5HashVisibility, Mode=OneWay}" />
+                    Visibility="{x:Bind ViewModel.Properties.ItemMD5HashVisibility, Mode=OneWay}" />
                 <TextBlock
                     x:Name="itemMD5HashValue"
                     Grid.Row="5"
                     Grid.Column="1"
                     Margin="20,0,0,0"
                     IsTextSelectionEnabled="True"
-                    Text="{x:Bind ItemProperties.ItemMD5Hash, Mode=OneWay}"
-                    Visibility="{x:Bind ItemProperties.ItemMD5HashVisibility, Mode=OneWay}" />
+                    Text="{x:Bind ViewModel.Properties.ItemMD5Hash, Mode=OneWay}"
+                    Visibility="{x:Bind ViewModel.Properties.ItemMD5HashVisibility, Mode=OneWay}" />
                 <ProgressBar
                     x:Name="ItemMD5HashProgress"
                     Grid.Row="5"
                     Grid.Column="1"
                     Margin="20,0,0,0"
-                    Visibility="{x:Bind ItemProperties.ItemMD5HashProgressVisibility, Mode=OneWay}" />
+                    IsIndeterminate="True"
```

Comment:
IsIndeterminate should be removed

Reply:
Resloved in this [commit ](https://github.com/VoidExploiter/files-uwp/commit/5c602a02968174eeb03bf84ec856261d4e85e837)